CMakeLists.txt

```diff
@@ -547,7 +547,25 @@ if(NOT MSVC)
   if(MOLD_SUPPORT)
     if(MOLD_EXECUTABLE)
       message(STATUS "Selecting mold as linker")
-      add_link_options("-fuse-ld=mold")
+      set(SELECT_MOLD_WITH_FUSE true)
```

Comment:
I'm not a cmake expert, maybe there's a smarter way to achieve the -fuse/-B switch without this bool.